application/org.openjdk.jmc.flightrecorder.flameview/src/main/resources/page.template

```diff
@@ -43,13 +56,21 @@
 				.call(flameGraph);
 		}
 
+		function executeSearch(){
```

Comment:
formatting: space before `{`

Reply:
maybe I've forgotten to run spotless, I'll do